src/components/stepforms/widgets/VariableInputs/VariableTag.vue

```diff
@@ -12,7 +12,6 @@
     <span class="widget-variable__tag-name">{{ variableLabel }}</span>
     <i
       class="widget-variable__tag-close fa fa-times"
-      @click="removeVariableTag"
       tabindex="1"
       @keypress.enter.prevent="removeVariableTag"
       @mousedown.prevent="removeVariableTag"
```

Comment:
Do you know why we need to use mousedown here rather than a click?

Reply:
To keep same behaviour that original multiselect one, it's more consistent!

Reply:
Ah ok! I didn't know multiselect was using these event. Thanks for the answer :100: